src/sentry/static/sentry/app/views/settings/account/accountNotificationFineTuning.jsx

```diff
@@ -95,9 +109,7 @@ class AccountNotificationsByProject extends React.Component {
 
   getFieldData() {
     const {projects, field} = this.props;
-    ProjectsStore.loadInitialData(projects);
```

Comment:
This caused `withProject()` to pass incorrect data into the organizationProjectsDashboard component later on.

Reply:
Was there anything else in account settings that depend on data from store?

Reply:
There are several calls to ProjectStore.getAll() and reflux listeners, but no other write operations I could find.